appsec/src/extension/backtrace.c

```diff
@@ -253,13 +261,21 @@ bool dd_report_exploit_backtrace(zend_string *nullable id)
         exploit = zend_hash_add_new(
             Z_ARR_P(dd_stack), _exploit_key, &EG(uninitialized_zval));
         array_init(exploit);
+        mlog(dd_log_trace, "Backtrace stack created");
     } else if (Z_TYPE_P(dd_stack) != IS_ARRAY) {
+        if (!get_global_DD_APPSEC_TESTING()) {
+            mlog(dd_log_trace,
+                "Field 'stack' is of type '%d', expected 'array'",
+                Z_TYPE_P(dd_stack));
```

Comment:
Additionally, this looks like an unexpected condition. Should be at least warning/error no?

Reply:
Changed